deps.edn

```diff
@@ -23,19 +23,21 @@
   com.layerware/hugsql-adapter-next-jdbc   {:mvn/version "0.5.1"}
   com.zaxxer/HikariCP                      {:mvn/version "5.0.0"
                                             :exclusions  [org.slf4j/slf4j-api]}
-  ;; Pedestal and Jetty webserver deps 
+  ;; Pedestal and Jetty webserver deps
   io.pedestal/pedestal.jetty               {:mvn/version "0.6.3"
                                             :exclusions  [org.eclipse.jetty.http2/http2-server]}
   org.eclipse.jetty.http2/http2-server     {:mvn/version "9.4.54.v20240208"}
   org.eclipse.jetty/jetty-alpn-java-server {:mvn/version "9.4.54.v20240208"}
   ;; Security deps
   buddy/buddy-core    {:mvn/version "1.11.418"
-                       :exclusions [org.bouncycastle/bcprov-jdk18on]}
+                       :exclusions [org.bouncycastle/bcprov-jdk18on
```

Comment:
I _was_ going to say you should also update buddy-core to the latest version of 1.11.423, but upon inspection of that version all it did was update both bouncycastle libs to 1.75, which still have the vulnerabilities.